src/types/OutList.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright (c) Whales Corp.
+ * All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+
+import { SendMode } from "./SendMode";
+import {loadMessageRelaxed, MessageRelaxed, storeMessageRelaxed} from "./MessageRelaxed";
+import {beginCell, Builder} from "../boc/Builder";
+import { Cell } from "../boc/Cell";
+import {Slice} from "../boc/Slice";
+import {loadStorageInfo, storeStorageInfo} from "./StorageInto";
+import {loadAccountStorage, storeAccountStorage} from "./AccountStorage";
+import {Account} from "./Account";
+import {loadMessage} from "./Message";
+
+export interface OutActionSendMsg {
+    type: 'sendMsg',
+    mode: SendMode;
+    outMsg: MessageRelaxed;
+}
+
+export interface OutActionSetCode {
+    type: 'setCode',
+    newCode: Cell;
+}
+
+export type OutAction = OutActionSendMsg | OutActionSetCode;
+
+export function storeOutAction(action: OutAction) {
+    if (action.type === 'sendMsg') {
+        return storeOutActionSendMsg(action);
+    }
+
+    return storeOutActionSetCode(action);
```

Comment:
If library is used on js, typo in code can lead to unexpected behavior

Reply:
fixed